.env

```diff
@@ -1,3 +1,2 @@
 PORT=8000
 REACT_APP_EPIC_SUPPORTED_QUERIES=false
-GENERATE_SOURCEMAP=false
```

Comment:
Just FYI, to make integrating upstream changes easier, this change may get removed if there are conflicts